webpack.dev.js

```diff
@@ -26,7 +26,7 @@ module.exports = merge(common, {
             loader: 'elm-webpack-loader',
             options: {
               verbose: true,
-              debug: true,
+              debug: !process.env.NO_ELM_DEBUGGER,
```

Comment:
i would prefer `CI_RUN` so we can make additional changes if running in the CI or not

Reply:
or create a different file just for the CI configuration that will be merged with the dev config

Reply:
A different file would be too much for such a small variation. However I'll take your advice and use `CI_RUN`, much better